po/extra/meson.build

```diff
@@ -0,0 +1,3 @@
+i18n.gettext('extra',
+    args: ['--directory=' + meson.source_root(), '--from-code=UTF-8']
```

Comment:
You need to make sure to set install to false here since the translations are compiled into the appdata file and the mo files are never used